cmd/lsif-go/main.go

```diff
@@ -50,14 +50,14 @@ func realMain() error {
 		return fmt.Errorf("either an output file using --out or --stdout must be specified")
 	}
 
-	if debug {
-		log.SetLevel(log.Debug)
-	}
-
 	if verbose {
 		log.SetLevel(log.Info)
 	}
 
+	if debug {
+		log.SetLevel(log.Debug)
+	}
```

Comment:
Quick question: was this change intentional? (i.e. show logs in debug level if both `verbose` and `debug` are `true`)

Reply:
`-verbose` will set Info, `-debug` will set Debug and will override the -verbose flag. The order they were meant if you do `-v -debug`, then you only get info output. Now specifying verbose and debug will take the most specific level.